cfme/common/vm.py

```diff
@@ -287,7 +287,7 @@ def get_detail(self, properties=None):
         return getattr(view.entities, properties[0].lower().replace(' ', '_')).get_text_of(
             properties[1])
 
-    def open_console(self, console='VM Console', invokes_alert=False, cancel=False):
+    def open_console(self, console='VM Console', invokes_alert=None):
```

Comment:
Pretty sure you want to default this to True, not None - None means no handling.  True means it'll confirm the alert that's raised when the console is opened.  If there is no alert during item_selection, the arg should be removed completely as the default for `item_select()` is `handle_alert=None`

https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L1489

Reply:
@kedark3 talked to me offline, and this is needed for webmks, and doesn't have alerts otherwise. Preferable to taking `**kwargs` too.